### PR TITLE
proposed fix for logout

### PR DESCRIPTION
**Pull request by mark-robustelli**

### CODE REVIEW

**Comment by mark-robustelli** [comment]
Incorporated some suggestions from [PR 6](https://github.com/FusionAuth/fusionauth-quickstart-python-django-web/pull/6).

File: complete-application/mysite/app/templates/account.html [hunk 1]
```diff
@@ -11,9 +11,9 @@
         <img src="https://fusionauth.io/cdn/samplethemes/changebank/changebank.svg" />
         <div class="h-row">
           <p class="header-email">{{ email }}</p>
-          <form id="logoutForm" class="button-lg"  action="{% url 'oidc_logout' %}" method="post">
+          <form action="{% url 'oidc_logout' %}" method="post">
             {% csrf_token %}
-            <a class="button-lg" href="#" onclick="document.getElementById('logoutForm').submit();">Logout</a>
+            <input type="submit" value="Logout" class="button-lg">
```
Comment (mark-robustelli) [inline complete-application/mysite/app/templates/account.html]: @hollygirouard , please help with the style for the submit button. It should match the Login anchor.

Comment (hollygirouard) [reply complete-application/mysite/app/templates/account.html]: @mark-robustelli Take a peep at this locally and let me know if you have any feedback. I just pushed it up. 
Screenshot 2024-06-18 at 10 58 40 AM

Comment (mark-robustelli) [reply complete-application/mysite/app/templates/account.html]: @hollygirouard Have you tried Chrome.  Still looks off.  
Screenshot 2024-06-18 at 12 47 20 PM

Comment (hollygirouard) [reply complete-application/mysite/app/templates/account.html]: Screenshot 2024-06-18 at 1 56 25 PM

Yes, try command + shift + R. Django likes to cache pretty hard.

Comment (mark-robustelli) [reply complete-application/mysite/app/templates/account.html]: @hollygirouard , That's the stuff.  I cleared my cache, at least I thought I did, but we are good now.  Thanks for the help!

Comment (hollygirouard) [reply complete-application/mysite/app/templates/account.html]: Woop woop! You're very welcome!

**Pull request merged**